core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -261,9 +261,12 @@ class LogCleaner(val config: CleanerConfig,
  * This class holds the actual logic for cleaning a log
  * @param id An identifier used for logging
  * @param offsetMap The map used for deduplication
- * @param bufferSize The size of the buffers to use. Memory usage will be 2x this number as there is a read and write buffer.
+ * @param ioBufferSize The size of the buffers to use. Memory usage will be 2x this number as there is a read and write buffer.
+ * @param maxIoBufferSize The maximum size of a message that can appear in the log
+ * @param dupBufferLoadFactor The maximum percent full for the deduplication buffer
  * @param throttler The throttler instance to use for limiting I/O rate.
  * @param time The time instance
+ * @param checkDone Check if the cleaning for a partition is aborted. If so, throw an exception
```

Comment:
I think the description for the checkDone param is somewhat misleading.

Notice that the Cleaner class uses checkDone but doesn't explicitly throw exceptions. It just so happens that the checkDone function passed into the Cleaner from the CleanerThread uses LogCleanerManager.checkCleaningAborted throws a LogCleaningAbortedException.